<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFTW_Parameter</class>
 <widget class="QWidget" name="FFTW_Parameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>683</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_TS_filename">
       <property name="text">
        <string>TSFileName</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ftime">
       <property name="text">
        <string>ftime</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ltime">
       <property name="text">
        <string>ltime</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_window">
       <property name="text">
        <string>FFT windows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_windows"/>
     </item>
     <item>
      <widget class="QLabel" name="label_fftlength">
       <property name="text">
        <string>FFT length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_length">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_freq">
       <property name="text">
        <string>Frequency Sheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_frefile">
       <property name="text">
        <string>Frequency File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_saveFreq">
       <property name="text">
        <string>Save Freq File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_freqtime">
       <property name="text">
        <string>Frequency Time Sheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fretimefile">
       <property name="text">
        <string>Frequency Time File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_saveFreqTime">
       <property name="text">
        <string>Save FreqTime File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget_fre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_fretime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_makeFFT">
     <property name="text">
      <string>Make FFT</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_freadd">
       <property name="text">
        <string>Fre Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fredel">
       <property name="text">
        <string>Fre Delet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_freall">
       <property name="text">
        <string>Fre Selet All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fretimeadd">
       <property name="text">
        <string>Freq Time Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fretimedel">
       <property name="text">
        <string>Freq Time Delet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fretimeall">
       <property name="text">
        <string>Freq Time Selet All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
